### PR TITLE
feat: filter options for event section

### LINKED ISSUE #8321: [BUG] Manage events toggle past events
### Description

When managing our events, the list gets long because of past events

- Add toggle or button to hide past events (hidden by default)

Screenshot 2023-07-23 at 11 19 22

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sital002**
## Fixes Issue

closes #8321 

## Changes proposed
filter option for event section
* past events
* future events
* all events
* paid events

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image
image

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Can it default to future events?

File: pages/api/account/manage/events.js [hunk 1]
```diff
@@ -49,6 +51,37 @@ export async function getEventsApi(username) {
...
+        dateEvents.push(cleanEvent);
+      } catch (e) {
+        log.error(e, `ERROR event date for: "${event.name}"`);
+      }
+    });
+
+    getEvents = dateEvents;
+  } else {
+    getEvents = [];
+  }
```
Comment (eddiejaoude) [inline pages/api/account/manage/events.js:84]: Is the `else` needed as `getEvents` is defined with `[]`?

Comment (sital002) [reply pages/api/account/manage/events.js:84]: ooh let me remove the else statement.

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**